### DISCUSSION TITLE
A Modern Spotify music player with python

### DISCUSSION THREAD

**Discussion by SarthakTools** [TomSchimansky/CustomTkinter#2492] [Showcase]
Hello TomSchimansky ,
I have created a modern Spotify music player app.
Please check it and give your feedback i spent a year making it ...
https://github.com/SarthakTools/Spotify

**Comment by BhagyaJyoti22006** [comment]
Oh loved it!

**Reply by SarthakTools** [reply]
Thank you so much brother !!!
